packages/nimbus/src/components/tabs/tabs.mdx

```diff
@@ -12,40 +12,355 @@ tags:
+
+#### Horizontal tabs
+
+Organize and display distinct, related sections of content immediately beneath a common navigation bar or header, serving primarily to group content contextually while conserving vertical screen space.
 
 ```jsx-live
 const App = () => (
-  <Tabs.Root size="md" defaultSelectedKeys={["tab1"]}>
+  <Stack direction="column" gap="600">
+    <Box>
+      <Text fontWeight="bold" mb="300">Large</Text>
+      <Tabs.Root size="lg">
+        <Tabs.List>
+          <Tabs.Tab id="general">General info</Tabs.Tab>
+          <Tabs.Tab id="history">History</Tabs.Tab>
+          <Tabs.Tab id="subscription">Subscription</Tabs.Tab>
+          <Tabs.Tab id="downloads">Downloads</Tabs.Tab>
+        </Tabs.List>
+        <Tabs.Panels pt="400">
+          <Tabs.Panel id="general">General information content goes here.</Tabs.Panel>
+          <Tabs.Panel id="history">History content goes here.</Tabs.Panel>
+          <Tabs.Panel id="subscription">Subscription content goes here.</Tabs.Panel>
+          <Tabs.Panel id="downloads">Downloads content goes here.</Tabs.Panel>
+        </Tabs.Panels>
+      </Tabs.Root>
+    </Box>
+
+    <Box>
+      <Text fontWeight="bold" mb="300">Medium</Text>
+      <Tabs.Root size="md">
+        <Tabs.List>
+          <Tabs.Tab id="general">General info</Tabs.Tab>
+          <Tabs.Tab id="history">History</Tabs.Tab>
+          <Tabs.Tab id="subscription">Subscription</Tabs.Tab>
+          <Tabs.Tab id="downloads">Downloads</Tabs.Tab>
+        </Tabs.List>
+        <Tabs.Panels pt="400">
+          <Tabs.Panel id="general">General information content goes here.</Tabs.Panel>
+          <Tabs.Panel id="history">History content goes here.</Tabs.Panel>
+          <Tabs.Panel id="subscription">Subscription content goes here.</Tabs.Panel>
+          <Tabs.Panel id="downloads">Downloads content goes here.</Tabs.Panel>
+        </Tabs.Panels>
+      </Tabs.Root>
+    </Box>
+
+    <Box>
+      <Text fontWeight="bold" mb="300">Small</Text>
+      <Tabs.Root size="sm">
+        <Tabs.List>
+          <Tabs.Tab id="general">General info</Tabs.Tab>
+          <Tabs.Tab id="history">History</Tabs.Tab>
+          <Tabs.Tab id="subscription">Subscription</Tabs.Tab>
+          <Tabs.Tab id="downloads">Downloads</Tabs.Tab>
+        </Tabs.List>
+        <Tabs.Panels pt="400">
+          <Tabs.Panel id="general">General information content goes here.</Tabs.Panel>
+          <Tabs.Panel id="history">History content goes here.</Tabs.Panel>
+          <Tabs.Panel id="subscription">Subscription content goes here.</Tabs.Panel>
+          <Tabs.Panel id="downloads">Downloads content goes here.</Tabs.Panel>
+        </Tabs.Panels>
+      </Tabs.Root>
+    </Box>
+  </Stack>
+)
+```
+
+#### Vertical tabs
+
+Maximize horizontal reading space and accommodate a larger number of consistently visible options than traditional horizontal tabs allow.
+
+```jsx-live
+const App = () => (
```

Comment:
The vertical orientation looks swapped to me?

Figma:
<img width="697" height="560" alt="Image" src="https://github.com/user-attachments/assets/c46f0ca1-c050-4b58-9cf5-0e76cb43d3bb" />
Docs:
<img width="650" height="263" alt="Image" src="https://github.com/user-attachments/assets/67d90b01-4e35-4b1d-90a3-54f0ef6e2fb1" />

Reply:
Aaaand a screenshot:
<img width="667" height="376" alt="image" src="https://github.com/user-attachments/assets/095dc0a8-075b-444f-8ad7-20df19de15b0" />